app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java

```diff
@@ -430,14 +432,19 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                     showSnackbar(mActivity, R.string.text_file_selected);
                     String fileName = mFileUtils.getFileName(mTextFileUri);
                     if (fileName != null) {
-                        if (fileName.endsWith(Constants.textExtension)) mFileExtension = Constants.textExtension;
-                        else if (fileName.endsWith(Constants.docxExtension)) mFileExtension = Constants.docxExtension;
-                        else if (fileName.endsWith(Constants.docExtension)) mFileExtension = Constants.docExtension;
+                        if (fileName.endsWith(Constants.textExtension))
+                            mFileExtension = Constants.textExtension;
+                        else if (fileName.endsWith(Constants.docxExtension))
+                            mFileExtension = Constants.docxExtension;
+                        else if (fileName.endsWith(Constants.docExtension))
+                            mFileExtension = Constants.docExtension;
                         else {
                             showSnackbar(mActivity, R.string.extension_not_supported);
                             return;
                         }
                     }
+                    String nameWithoutExtension = mFileUtils.stripExtension(fileName);
+                    mSelectedFileName = nameWithoutExtension + getString(swati4star.createpdf.R.string.pdf_suffix);
                     fileName = getString(R.string.text_file_name) + fileName;
```

Comment:
we already have this variable `fileName`
why dont you use this only, instead of creating a new variable `mSelectedFileName`

Reply:
fileName is modified to "Text File: Filename" for setting it in mSeletFile textview and also it is a local variable. because of that, it is not accessible in materialdialogBuilder scope. So,  I had to create a new variable for holding just the filename_pdf. 
and plz note, this line is wrong:
`fileName = getString(R.string.text_file_name) + fileName;`
adding string to a well-defined variable gives it altogether a different meaning

Suggestion: 
`mSelectFile.setText(getString(R.string.text_file_name) + fileName);`
OR
`String displayText = getString(R.string.text_file_name) + fileName; 
  mSelectFile.setText(displayText);`